### DISCUSSION TITLE
Changing username to my second accounts name

### DISCUSSION THREAD

**Discussion by zsoolti8917** [community/community#53696] [Other Feature Feedback, Questions, & Ideas]
Hello, i would like to discuss if it is possible to change my current GitHub accounts username to the username which my second account has @zsoltvarju (i have acces to both). The one which i have now is not very professional and my name is what the other account has.

My first idea was to delete the second account but according to the pop-up the username will be held for 90 days:
![image](https://user-images.githubusercontent.com/50336024/234312391-4d179f55-46f5-479c-bc08-441556605c6f.png)

My problem is that i have a fear that someone will take it, plus i started to intensively use GitHub for various projects and use @zsoolti8917 for mentioning my github in various pages. The later i get the new username the more links i will need to refactor.

**Comment by 90th** [comment]
Simply log into both accounts, and change the @zsoltvarju accounts username to something different, and quickly switch the main account you use now to the username you wanted "@zsoltvarju" i doubt someone will snipe that username under the few secs it takes you to swap both of them.

I have never changed my username before so im not 100% sure how it works.

**Comment by naveen-naga** [comment]
> Hello, i would like to discuss if it is possible to change my current GitHub accounts username to the username which my second account has @zsoltvarju (i have acces to both). The one which i have now is not very professional and my name is what the other account has.
> 
> My first idea was to delete the second account but according to the pop-up the username will be held for 90 days: ![image](https://user-images.githubusercontent.com/50336024/234312391-4d179f55-46f5-479c-bc08-441556605c6f.png)
> 
> My problem is that i have a fear that someone will take it, plus i started to intensively use GitHub for various projects and use @zsoolti8917 for mentioning my github in various pages. The later i get the new username the more links i will need to refactor.

Hey I'm facing the same problem,
Did you rename those account?
If yes, Did you face any issues while renaming?

**Comment by wasokeli** [comment]
There's a 90-day holding period when you delete an account, but there's no holding period when you change the username of an account.

I swapped the names of my new and old accounts, and I was able to do so immediately.
